compiler/rustc_interface/src/interface.rs

```diff
@@ -347,7 +356,7 @@ pub struct Config {
     /// enabled. Makes it so that "please report a bug" is hidden, as ICEs with
     /// internal features are wontfix, and they are usually the cause of the ICEs.
     /// None signifies that this is not tracked.
-    pub using_internal_features: Arc<std::sync::atomic::AtomicBool>,
+    pub using_internal_features: &'static std::sync::atomic::AtomicBool,
```

Comment:
The downside of this static lifetime is that it becomes a bit annoying to rerun the compiler from the same process as you need to reset that bool

Reply:
This is effectively a global anyway given that it is returned by `install_ice_hook` which replaces the panic hook for the entire process.